pkg/snapstore/abs_snapstore.go

```diff
@@ -145,24 +147,13 @@ func NewABSSnapStore(config *brtypes.SnapstoreConfig) (*ABSSnapStore, error) {
 
 // ConstructBlobServiceURL constructs the Blob Service URL based on the activation status of the Azurite Emulator.
 // It checks the environment variable for emulator configuration and constructs the URL accordingly.
```

Comment:
Can this comment, line 147 be removed/changed to reflect the changes?

Reply:
Sure, will do